conditional/blueprints/dashboard.py

```diff
@@ -58,7 +58,8 @@ def display_dashboard():
     c_meetings = [m.meeting_id for m in
                   MemberCommitteeAttendance.query.filter(
                       MemberCommitteeAttendance.uid == member.uid
-                  )]
+                  ) if CommitteeMeeting.query.filter(
```

Comment:
I am not sure the sacrifice in readability is worth the four or five milliseconds you would save on load time. I looked back at the query I wrote for the housing queue and had to turn my monitor upside just trying to remember why I did what I did.